shared/common-adapters/markdown/shared.js

```diff
@@ -77,7 +77,7 @@ function createMentionRegex(meta: ?MarkdownMeta): ?RegExp {
 }
 
 function createChannelRegex(meta: ?MarkdownMeta): ?RegExp {
-  if (!meta || !meta.mentionsChannelName) {
+  if (!meta || !meta.mentionsChannelName || meta.mentionsChannelName.isEmpty) {
```

Comment:
`.isEmpty` -> `.isEmpty()`

Reply:
nice catch! I'll fix the clickable thing in another branch

Reply:
O actually, this might fix your clickable issue